### ISSUE TITLE
Error: ‘ggml_flash_attn’ was not declared in this scope; did you mean ‘ggml_flash_attn_ext’?

### ISSUE THREAD

**Issue by wr131** [leejet/stable-diffusion.cpp#456]
I complied this repositories on my Debian 12 PC,but it was failed. 
error:

    error: ‘ggml_flash_attn’ was not declared in this scope; did you mean ‘ggml_flash_attn_ext’?
      681 |     struct ggml_tensor* kqv = ggml_flash_attn(ctx, q, k, v, false);  // [N * n_head, n_token, d_head]
          |                               ^~~~~~~~~~~~~~~
          |                               ggml_flash_attn_ext'

parameters:
'DGGML_AVX2 off SD_FLASH_ATTN on CXX_FLAGS=-march=ivybridge'

**Comment by Green-Sky** [comment]
Yes, don't use `SD_FLASH_ATTN`. It stopped working early this year.

**Comment by truedat101** [comment]
yep

**Comment by leejet** [comment]
Please use the latest code and update the ggml submodule.

**Comment by truedat101** [comment]
Thanks, will try tonight.
